person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :age, :name
+
+  def initialize(age, name)
+    @id = DateTime.now
+    @name = name
+    @age = age
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age?
+  end
```

Comment:
- [ ] Hi @jheart-vic, based on the requirements of this project, this public method(can_use_services?) should return true if a person is of age :+1: or if they have permission from parents(`is_of_age?() || @parent_permission`) :angry:  
> **Quick note**
> > You'll equally need to set `parent_permission` as a parameter in the constructor above and default the value to `true` that's 
```ruby   
   def initialize(age, name =  "Unknown", parent_permission = true)
    
      @parent_permission = parent_permission
   end
```

[ReadMore...](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars)

Thanks :pray: